### ISSUE TITLE
Feature idea - provisioning registrations

### ISSUE THREAD

**Issue by variable** [CZ-NIC/django-fido#210]
I have a feature request from my workplace to add an approval step for self-registering keys. Currently when someone registers a key, it can be authenticated immediately, the `Authenticator` model doesn't have a status field to control disable the key.

Is this something I can implement and be included into this package?

**Comment by tpazderka** [comment]
This can be currently (somewhat) achieved by disabling self-registration and using an admin view to add `Authenticator` to selected users.

As for the proposed change, I am not opposed to it as long as it is an opt-in.

@ziima @stinovlas your thought?

**Comment by ziima** [comment]
Sounds reasonable

**Comment by stinovlas** [comment]
I think this might prove useful in our workplace as well. Being able to disable authenticator without removing it sound like a good feature anyway =o).

**Comment by variable** [comment]
> This can be currently (somewhat) achieved by disabling self-registration and using an admin view to add `Authenticator` to selected users.
> 
> As for the proposed change, I am not opposed to it as long as it is an opt-in.
> 
> @ziima @stinovlas your thought?

Yeah this is what we have done so far, I have disabled self-registration but work is floating an idea to re-enable it again with approval step.
